NetKAN/Kazza197sNukeAddonToBahamutoDsArmoury.netkan

```diff
@@ -0,0 +1,15 @@
+{
+    "spec_version": 1,
+    "license": "CC-BY-4.0",
```

Comment:
`unknown` or `unrestricted` are probably better choices for the license for this one, although I'm not sure what technically qualifies as "unrestricted". @pjf ?